### PR TITLE
CHANGELOG: release v0.107.2

### PR DESCRIPTION
**Pull request by AnnaShaleva**
Depends on #3725, #3749, #3750, #3751.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,45 @@
...
+of BlockFetcher service that may hang on retry of NeoFS requests preventing the node
+from syncing. Also, an improved algorithm of blocks uploading and extended list of
+block and index file attributes are supported for `upload-bin` CLI command.
+
+No configuration update or DB resync is required. However, starting from this release
+`NeoFSBlockFetcher`application configuration section is backed by default values for
+every parameter except `Addresses` and `ContainerID`, hence if you don't like too
+chatty configuration files, feel free to remove all optional parameters. Also, for
+those who'd like to experiment with untraceable data removal, enable
+`RemoveUntraceableHeaders` setting in the `ApplicatoinConfiguration` config section.
```
Comment (roman-khimov) [inline CHANGELOG.md:20]: Application

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,45 @@
...
+an experimental mode, use it with care. Other than that, this release includes a fix
+of BlockFetcher service that may hang on retry of NeoFS requests preventing the node
+from syncing. Also, an improved algorithm of blocks uploading and extended list of
+block and index file attributes are supported for `upload-bin` CLI command.
+
+No configuration update or DB resync is required. However, starting from this release
+`NeoFSBlockFetcher`application configuration section is backed by default values for
+every parameter except `Addresses` and `ContainerID`, hence if you don't like too
+chatty configuration files, feel free to remove all optional parameters. Also, for
+those who'd like to experiment with untraceable data removal, enable
```
Comment (roman-khimov) [inline CHANGELOG.md:19]: Maybe not worth reiterating.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,45 @@
...
+One more compatible patch-release that introduces `RemoveUntraceableHeaders`
+application-level extension allowing to remove untraceable block headers from the DB.
+This feature significantly reduces the database size, but for now it is supported in
+an experimental mode, use it with care. Other than that, this release includes a fix
+of BlockFetcher service that may hang on retry of NeoFS requests preventing the node
+from syncing. Also, an improved algorithm of blocks uploading and extended list of
+block and index file attributes are supported for `upload-bin` CLI command.
+
+No configuration update or DB resync is required. However, starting from this release
+`NeoFSBlockFetcher`application configuration section is backed by default values for
```
Comment (roman-khimov) [inline CHANGELOG.md:16]: Missing space before `application`.

**Comment by AnnaShaleva** [comment]
Waiting for SDK fixes.

File: CHANGELOG.md (hunk 2 above)
Comment (AnnaShaleva) [reply CHANGELOG.md]: Removed.

File: docs/node-configuration.md [hunk 4]
```diff
@@ -185,27 +185,29 @@ where:
 - `UnlockWallet` contains wallet settings to retrieve account to sign requests to
   NeoFS. Without this setting, the module will use randomly generated private key.
   For configuration details see [Unlock Wallet Configuration](#Unlock-Wallet-Configuration)
-- `Addresses` is a list of NeoFS storage nodes addresses.
-- `Timeout` is a timeout for a single request to NeoFS storage node.
-- `ContainerID` is a container ID to fetch blocks from.
+- `Addresses` is a list of NeoFS storage nodes addresses. This parameter is required.
+  for BlockFetcher functioning.
```
Comment (AliceInHunterland) [inline docs/node-configuration.md:189]: `NeoFS BlockFetcher` but at the same time in `network/server` we do have `blockFetcher`.

Comment (AnnaShaleva) [reply docs/node-configuration.md:189]: `NeoFS BlockFetcher` is a name of the service. `blockFetcher` is a name of unexported field.

Comment (AnnaShaleva) [reply docs/node-configuration.md:189]: Everywhere in the documentation it's `BlockFetcher` If it's not, then documentation must be fixed.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**